### DISCUSSION TITLE
VSCode Copilot extension not loading

### DISCUSSION THREAD

**Discussion by zaibon** [community/community#60212] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Copilot Version: `v1.95.233`
VSCode version: 
```
Version: 1.79.2 (Universal)
Commit: 695af097c7bd098fbf017ce3ac85e09bbc5dda06
Date: 2023-06-14T08:58:52.392Z
Electron: 22.5.7
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.5.0
```

The extension does not seem to even load anymore. I do not see it in the output panel. Impossible to execute any command from the pallet either.

<img width="202" alt="Screenshot 2023-07-06 at 8 21 23 AM" src="https://github.com/community/community/assets/303164/4f1714d9-2d45-4c06-ad06-419d960bb74a">
<img width="243" alt="Screenshot 2023-07-06 at 8 21 56 AM" src="https://github.com/community/community/assets/303164/cdc07e6e-4d86-4e35-a087-f75f2226d3a2">

**Comment by Akash1134** [comment]
Hi @zaibon , Thanks for reporting the issue and sorry for the trouble caused. Could you please reinstall VSCode and Copilot extension? (This is not a great first step but the error says some components are missing and this might help resolve the issue). 🙂

**Reply by zaibon** [reply]
Hi @Akash1134,

What are the folders that needs to be cleaned up on MacOS to completely remove VSCode?
Because just deleting the application and re-installing it does not change anything. I'm suspecting it's because it still had all the local config somewhere.

**Comment by prog8** [comment]
I also can see similar behavior and it seems to be related with another extensions interfering with Copilot. Do you know what could be a reason?

**Comment by zaibon** [comment, marked as answer]
I have found the root cause of my problem.
I had another extension that was doing something not right and prevented the Copilot from properly initializing. 
Fixing the faulty extension resolved my issue.
